AppleStore/AppleStore/PageViewController/WomanHealthPageViewController.swift

```diff
@@ -0,0 +1,161 @@
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - LifeCycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setVisualComponents()
+        setBasesComponents()
+    }
+
+    // MARK: - Private Methods
+    private func setBasesComponents() {
+        createWomanHealthPage()
+        createPageControl()
+        dataSource = self
+        delegate = self
+    }
+
+    private func setVisualComponents() {
+        createButton(nextButton, name: "NEXT", axisX: Int(UIScreen.main.bounds.width) - 160)
+        nextButton.addTarget(self, action: #selector(tapToNextButton), for: .touchUpInside)
+
+        createButton(skipButton, name: "SKIP", axisX: 0)
+        skipButton.addTarget(self, action: #selector(tapToSkipButton), for: .touchUpInside)
+
+        createButton(startButton, name: "Get Started!", axisX: Int(UIScreen.main.bounds.width) / 2 - 80)
+        startButton.addTarget(self, action: #selector(tapToSkipButton), for: .touchUpInside)
+        startButton.isHidden = true
+    }
+
+    private func createButton(_ param: UIButton, name: String, axisX: Int) {
+        param.frame = CGRect(x: axisX, y: Int(UIScreen.main.bounds.height) - 100, width: 160, height: 50)
+        param.setTitle(name, for: .normal)
+        param.titleLabel?.font = UIFont(name: "BodoniSvtyTwoITCTT-Bold", size: 20)
+        param.setTitleColor(.systemBlue, for: .normal)
+        view.addSubview(param)
+    }
+
+    private func createWomanHealthPage() {
+        let trackCyclePage = WomanHealthHelper(nameImageText: "TrackImage",
+                                               titleText: "Track Your Cycle",
+                                               descriptionText: "Manage irregular period and learn how to improve your period.")
+        let planPregnancyPage = WomanHealthHelper(nameImageText: "PlanImage",
+                                                  titleText: "Plan Your Pregnancy",
+                                                  descriptionText: "Favorable days are important. Vestibulum rutrum quam vitae fringilla tincidunt.")
+        let dailyHealthPage = WomanHealthHelper(nameImageText: "DailyImage",
+                                                titleText: "Daily Health Insight",
+                                                descriptionText: "Personal health insight. Vestibulum rutrum quam vitae fringilla tincidunt.")
+        womanHealthPages.append(trackCyclePage)
+        womanHealthPages.append(planPregnancyPage)
+        womanHealthPages.append(dailyHealthPage)
+    }
+
+    private func createPageControl() {
+        let pageControllAppearance = UIPageControl.appearance()
+        pageControllAppearance.pageIndicatorTintColor = .lightGray
+        pageControllAppearance.currentPageIndicatorTintColor = .systemBlue
+    }
+
+    private func checkPage(page: Int) {
```

Comment:
```suggestion
    private func checkPage(page: Int) {
```
в названии checkPage чек а в теле hidden не верное название значит